content/en/docs/reference/command-line-tools-reference/feature-gates/service-account-token-node-binding.md

```diff
@@ -13,5 +13,9 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.31"
+    toVersion: "1.32"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.33"
```

Comment:
This file has been renamed in the 'main' branch.
Please consider rename it accordingly so that there won't be conflicts when 'main' is merged to 'dev-1.33'.

Reply:
I'll let the periodic sync of main → dev-1.33 handle that, and rebase once that is done